frontend/src/shared/components/checkbox-component/checkbox.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+@Component({
+  selector: 'app-checkbox',
+  templateUrl: './checkbox.component.html',
+  styleUrls: ['./checkbox.component.sass']
+})
+export class CheckboxComponent implements OnInit {
+  @Input() public Text = '';
```

Comment:
use lowercase for fields and variables, please

Reply:
Fixed. Used lowercase